### DISCUSSION TITLE
添加云盘只有4种

### DISCUSSION THREAD

**Discussion by ShuX503** [AlistGo/alist#2054] [Invalid question]
只能添加阿里、天冀、移动、123这4个网盘

**Comment by ShuX503** [comment]
是不是哪儿设置不对哦

**Comment by anwen-anyi** [comment]
你下载错版本了吧似乎是.....

**Reply by ShuX503** [reply]
alist-windows-4.0-amd64.zip，我下的这一个呀

**Comment by LovelyTitan** [comment]
百度也可以的呀

**Reply by ShuX503** [reply]
就是没有哇，你下的哪个安装包呀？

**Comment by LovelyTitan** [comment]
我是通过群晖应用商店里下的，版本v3.2.0




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年10月19日(星期三) 晚上10:58
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] 添加云盘只有4种 (Discussion #2054)





 
就是没有哇，你下的哪个安装包呀？

**Reply by ShuX503** [reply]
我是windows，也没群晖应用商店呀

**Comment by WindProtect** [comment]
那个。。。要不，你试下把鼠标移动到下面的列表后，用中间的滚轮滚动一下就能看到别的了？或者换个浏览器试下？

**Reply by ShuX503** [reply]
还真是，谢谢谢谢！
